scripts/azure-pipelines/linux/provision-image.sh

```diff
@@ -58,16 +58,16 @@ cd ~
 curl -sSL https://get.haskellstack.org/ | sudo sh
 
 # Install CUDA
-wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-ubuntu1804.pin
+wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-ubuntu2004.pin
 sudo mv cuda-ubuntu1804.pin /etc/apt/preferences.d/cuda-repository-pin-600
```

Comment:
I'm not sure why it seemed to work with this broken...

```suggestion
sudo mv cuda-ubuntu2004.pin /etc/apt/preferences.d/cuda-repository-pin-600
```